java-agent/java-agent.gradle

```diff
@@ -22,14 +22,14 @@ jar {
       "Premain-Class": "io.opentelemetry.auto.bootstrap.AgentBootstrap",
       "Can-Redefine-Classes": true,
       "Can-Retransform-Classes": true,
-    )
+      )
   }
 }
 
 CopySpec isolateSpec(Collection<Task> sourceTasks) {
   return copySpec {
     from(sourceTasks.collect { zipTree(it.archiveFile) }) {
-      into 'auto-tooling-and-instrumentation.isolated'
+      into 'inst'
```

Comment:
Why this change?

Reply:
I suggest you review the original PR.

Reply:
Link? :)

Reply:
i added comment

also, the original PRs are all linked in the corresponding commit messages